packages/inter-protocol/src/price/roundsManager.js

```diff
@@ -101,12 +101,14 @@ export const prepareRoundsManagerKit = baggage =>
       contract: M.interface(
         'contract',
         {
-          authenticateQuote: M.call(M.any()).returns(M.any()),
-          makeCreateQuote: M.call().optional(M.any()).returns(M.any()),
-          eligibleForSpecificRound: M.call(M.any()).returns(M.boolean()),
-          getRoundData: M.call(M.any()).returns(M.promise()),
-          getRoundStatus: M.call(M.any()).returns(M.record()),
-          oracleRoundStateSuggestRound: M.call(M.any()).returns(M.record()),
+          authenticateQuote: M.call().rest(M.any()).returns(M.any()),
```

Comment:
This should take one quote,
https://github.com/Agoric/agoric-sdk/blob/a271af98a48928e151404e622b64f4ce808e86f0/packages/zoe/tools/manualPriceAuthority.js#L39-L45

I'll push accurate guards.

Reply:
Better guards would certainly be nice, but can happen after this PR. Thus, I won't withhold approval waiting for this conversation to be resolved.